pages/linux/procs.md

```diff
@@ -0,0 +1,25 @@
+# procs
+
+> Displays information about the active processes.
+> It can be a used as ps or top alternative.
```

Comment:
```suggestion
```
Needless

Reply:
Not directly needless, but also not really important.